### ISSUE TITLE
Update license to GPL2

### ISSUE THREAD

**Issue by MrSapps** [q-gears/q-gears#24]
zerotag has added some files with the MIT license, this must be changed to GPL, and the github license file must be updated to GPL.

**Comment by paulsapps** [comment]
bgroff - based on your recent pull request you might wanna take this one?

**Comment by bgroff** [comment]
Yeah I can take this. Do we want to add copyleft notice in the top of cpp files? 

Also I think that the project is under the GPL, but is everyone set on that? MIT/BSD is a more permissive license but just wanted to get opinions.
